src/app/components/shared/datetime/datetime/datetime.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input } from "@angular/core";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { TimezoneInformation } from "@interfaces/apiInterfaces";
+import { DateTime, IANAZone, Zone } from "luxon";
+import { AbstractDatetimeComponent } from "../abstract-datetime.component";
+
+type BawTimezoneUnion = Zone | TimezoneInformation | string;
+
+/**
+ * Displays a datetime (possibly from another timezone) in the users locale and timezone
+ * Commonly used for displaying client side timestamps e.g. Project created at: xyz
+ *
+ * This component does not assume that all dates input are utc, and will localize using ISO8601 offset information if possible
+ */
+@Component({
+  selector: "baw-datetime",
+  templateUrl: "../abstract-datetime.component.html",
+})
+export class DatetimeComponent extends AbstractDatetimeComponent {
+  public constructor() {
+    super();
+  }
+
+  @Input({ required: true }) public value!: DateTime | Date;
+  @Input() public dateOnly: string | boolean;
+  @Input() public timeOnly: string | boolean;
+  protected timezone?: BawTimezoneUnion;
```

Comment:
why is this here? it makes no sense...

Reply:
I've moved all timezone logic to the `zoned-datetime.component.ts`

This was originally here because the `datetime.component.ts` needs to have knowledge of the users local timezone.

Therefore, I made the `zoned-datetime.component.ts` and `datetime.component.rs` have the same timezone logic. The `value` (fetch from `luxonDateTime`) was localized, meaning that it was actually just using the implicit timezone logic when creating the tooltip, using the timezone, etc...

Essentially the only difference between the `zoned-datetime.component.ts` and `datetime.component.ts` was that you could overwrite the timezone localization using the `timezone` `@Input()` (if it was null it'd localize to the users local timezone)

Reply:
I understand your reasoning but from a functional composition perspective you still had one type import and store a property of a type it itself could never use.